src/main/java/duke/ui/Launcher.java

```diff
@@ -0,0 +1,17 @@
+package duke.ui;
+
+import duke.Main;
+import javafx.application.Application;
+
```

Comment:
according to your ordering of import statements, should there be an empty line between the first import statement and the next :D?